delegation-toolkit/concepts/delegation/caveat-enforcers.md

```diff
@@ -119,23 +119,23 @@ This "all-or-nothing" approach ensures that delegations only execute exactly as
 
 ## Caveat builder
 
-While caveat enforcers operate at the smart contract level, most developers interact with them through the [`CaveatBuilder`](../guides/create-delegation/restrict-delegation.md) interface in the MetaMask Delegation Toolkit.
+While caveat enforcers operate at the smart contract level, most developers interact with them through the [`CaveatBuilder`](../../guides/delegation/restrict-delegation.md) interface in the MetaMask Delegation Toolkit.
```

Comment:
Just FYI, this is not true anymore. In the new version it won't be required. Delegations will have `ScopeConfig` and caveats are declarative. We can merge the changes, I'll update with v0.13 changes.